scss/components/_control-bar.scss

```diff
@@ -169,8 +149,26 @@
 }
 
 .oo-interactive-container {
+  width: 100%;
+
   .oo-control-bar.oo-control-bar-hidden {
     height: 0 !important;
     opacity: 0;
   }
 }
+
+.oo-control-bar-video {
+  background: $control-bar-bg;
+  @include vendor-prefixes (background, $control-bar-gradient);
+  @include vendor-prefixes (transition, $medium-transition);
+  height: $control-bar-height;
+  padding: 0 15px;
+
+  .oo-control-bar-items-wrapper {
+    margin-top: 10px;
+
+    & > * {
```

Comment:
Is it possible to replace the `*` operator with something more explicit? If we add different elements to this container in the future they will inherit this style automatically and might be hard to debug

Reply:
I'm not sure, this was part of the original control bar styling. I copied this from https://github.com/ooyala/html5-skin/pull/1126/files/c5ea5e71f2fd39b8872ef99dc7df98a39d662f10#diff-256e213b9742bfdba7b41aa6315beb3eL32. I'd prefer not to change this in this PR, but we should look into this in the future.